flexmeasures/ui/crud/users.py

```diff
@@ -76,18 +76,15 @@ def process_internal_api_response(
 
 def get_users_by_account(
```

Comment:
I believe that function is now not used at all anymore, so it can go and also the test.